frontend/src/components/login_page/login.css

```diff
@@ -0,0 +1,161 @@
+.login-page{
+    background-color: #201E1F;
```

Comment:
To make it easier to reference colors in css, you should use css variables

It will:
 1. it makes your css more readable
 2. it's more convenient to use the color name instead of copy and pasting color hexcode every single time
 3. it makes it easier to replace any of your app color throughout the app

**step 1:**
To do this you can declare a `:root` psuedoclass at the very top of your css file and add colors in like this:

```css
:root {
    --black: ##201E1F;
    --green: #06D5A0;
    --blue: #1B9AAA;
}
```
Note: personally, i would add in the top-level like `app.css` or `index.css` so that it's easier to find and configure.

**step 2:**
You can use these variables throughout your CSS by referencing them with the var() function. For example:
```css
.login-page {
     background-color: var(--black);
     ...other properties
}
```